docs/tutorials/aws.md

```diff
@@ -79,6 +76,10 @@ You can use Attribute-based access control(ABAC) for advanced deployments.
 }
 ```
 
+### Further improvements
+
+Both policies can be further enhanced by tightening them down. Following the [principle of least privelege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege) and explicitely providing a list of selected zones instead of `*` you can scope the deployment down allowing changes only to zones from the list hence reducing the blast radius and improving auditability.
```

Comment:
```suggestion
Both policies can be further enhanced by tightening them down. Following the [principle of least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege) and explicitly providing a list of selected zones instead of `*` you can scope the deployment down allowing changes only to zones from the list hence reducing the blast radius and improving auditability.
```